lib/src/base-styles/_nx-colors.scss

```diff
@@ -293,6 +293,10 @@
   --nx-color-status-indicator-error-icon-background: var(--nx-swatch-red-65);
   --nx-color-status-indicator-error-icon-border: var(--nx-color-alert-error);
 
+  // NxColorPicker
+  --nx-color-color-picker-border-focus: var(--nx-swatch-teal-35);
```

Comment:
A seeing a discrepancy in the mocks: at least one spot has the focus ring as blue-40, though most have it as teal-35